models/topics.models.js

```diff
@@ -0,0 +1,10 @@
+const { topicsData } = require("../db/data/test-data");
+const db = require("../db/connection.js");
+
+exports.fetchTopicData = () => {
+  let query = `SELECT * FROM topics;`;
+
+  return db.query(query).then((res) => {
+    return res.rows;
+  });
+};
```

Comment:
Once again, `fetchTopicData` is mostly a well named variable. And the database query string and `.query` method are implemented well. It's good to see you have not set up a `catch` block here as it is not needed, as the controller will deal with any errors caught, well done Cristina!